README.md

```
@@ -1 +1,195 @@
 # github-personal-statistics
+
+個人のGitHub活動量を計測し、Slackに通知するツール。月次の目標に対する進捗を日次で追跡します。
+
+## 機能概要
+
+### 計測指標
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)